diceRoll/diceRollUnitTests.js

```diff
@@ -98,87 +93,87 @@ function testList() {
       app.toggleMenu = new Stub();
       app.userSaveButtonCheckDisplay = new Stub();
       app.run();
-      assert.compare(app.addRollBarListeners.info.called, 1, 'addRollBarListeners() not called');
-      assert.compare(app.addCalculatorListeners.info.called, 1, 'addCalculatorListeners() not called');
-      assert.compare(app.addUserSaveButtonListener.info.called, 1, 'addUserSaveButtonListener() not called');
-      assert.compare(app.simulateFirstVisit.info.called, 1, 'simulateFirstVisit() not called');
-      assert.compare(app.checkMemory.info.called, 1, 'checkMemory() not called');
-      assert.compare(app.toggleMenu.info.called, 1, 'toggleMenu() not called');
-      assert.compare(app.userSaveButtonCheckDisplay.info.called, 1, 'userSaveButtonCheckDisplay() not called');
+      assert.compare(app.addRollBarListeners.info.calls.length, 1, 'addRollBarListeners() not called');
```

Comment:
Now change the name of this test to:

"app.run() starts the app by initializing all the event listeners, loading data into memory, and showing the initial screen/menus"

That description will also make you think about the names of the methods that you are calling and consider renaming them to fit what they actually do

Reply:
(Talking about simulateFirstVisit and checkMemory)